wave-api/src/main/java/io/seqera/wave/api/ContainerConfig.java

```diff
@@ -50,12 +53,13 @@ public ContainerConfig() {
         layers = new ArrayList<>();
     }
 
-    public ContainerConfig(List<String> entrypoint, List<String> cmd, List<String> env, String workDir, List<ContainerLayer> layers) {
+    public ContainerConfig(List<String> entrypoint, List<String> cmd, List<String> env, String workDir, List<ContainerLayer> layers, Map<String, String> labels) {
         this.entrypoint = entrypoint;
         this.cmd = cmd;
         this.env = env;
         this.workingDir = workDir;
         this.layers = layers;
+        this.labels = labels;
```

Comment:
shouldn't the the default (no args) initialiser above also define an empty `labels` object?
i.e. line 54
```
labels = new HashMap<>();
```

Reply:
done